src/components/CheckHealth.test.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { CheckHealth } from './CheckHealth';
+import { InstanceContext } from './InstanceContext';
+import { Check } from 'types';
+import { getInstanceMock, instanceSettings } from '../datasource/__mocks__/DataSource';
+
+const defaultCheck = {
+  id: 2,
+  tenantId: 1,
+  frequency: 60000,
+  offset: 0,
+  timeout: 2500,
+  enabled: true,
+  labels: [],
+  settings: {
+    ping: {
+      ipVersion: 'V4',
+      dontFragment: false,
+    },
+  },
+  probes: [1],
+  target: 'grafana.com',
+  job: 'tacos',
+  created: 1597928927.7490728,
+  modified: 1597928927.7490728,
+} as Check;
+
+const renderCheckHealth = (check: Check = defaultCheck) => {
+  const instance = {
+    api: getInstanceMock(instanceSettings),
+  };
+  return render(
+    <InstanceContext.Provider value={{ instance, loading: false }}>
+      <CheckHealth check={check} />
+    </InstanceContext.Provider>
+  );
+};
+
+test('renders without crashing', async () => {
```

Comment:
I'm not stoked on either these queries or the assertions, but I don't see any accessible ways to assert a type of icon